app/presenters/vendor_api/single_application_presenter.rb

```diff
@@ -18,10 +18,7 @@ def as_json
             first_name: application_form.first_name,
             last_name: application_form.last_name,
             date_of_birth: application_form.date_of_birth,
-            nationality: [
-              application_form.first_nationality,
-              application_form.second_nationality,
-            ].compact,
+            nationality: %w[NL],
```

Comment:
How does this sit with https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/346/commits/f38470af82ac017f7e31f4f97ec2ec5fba154784?

Reply:
I had tried to get rid of the hard-coded value but backed out because it had an impact on a bunch of other specs. After chatting to @tijmenb I think I'm going to have another go at the spec for this one so will revisit.